Save the performance log to file

yhirokawa-ccs opened the pull request:
1. Hamiltonian GFLOPS
    - Processor 0
    - Processor (best performance in allocated nodes)
    - Single-cell (in multi-scale mode only)
    - System (summation of allocated nodes)
2. load-balance check in dynamics part
    - minimum execution time [sec]
    - maximum execution time [sec]
    - difference (max - min)
    - relative difference (max / min)

File name format: `(sc/ms)_performance_YYYYMMDD_hhmmss.log`

If you read the file, you execute the below command in jobscript.
`ls -t data/*performance*log | head -n 1 | xargs cat`

yhirokawa-ccs committed:
Save the performance log to file.

shunsuke-sato commented on modules/performance_analyzer.f90:
```diff
@@ -43,44 +48,122 @@ subroutine get_hamiltonian_performance(lgflops)
+
+    if (Myrank == 0) then
+      write (iounit,'(A)') 'Load balance check [sec]'
+      write (iounit,'(A,4(A12))') 'Function    ','min','max','diff','rel [%]'
```
Thank you for implementing the load balance checker! I have a comment. Probably, the value of "rel()" should be multiplied by a factor of 100 to be described as a percentage.

yhirokawa-ccs commented:
@shunsuke-sato san, thank you for review.
I made a mistake the unit of relative performance. I removed it.
And, I squashed the commits of pull request.
